test/common/config_parser_helper.erl

```diff
@@ -1102,7 +1102,7 @@ extra_service_listener_config() ->
       state_timeout => 5000,
       hidden_components => false,
       conflict_behaviour => disconnect,
-      connection_type => component}.
+      connection_type => undefined}.
```

Comment:
connection_type is undefined (was `component`). Is it intentional?

Reply:
Yes, it comes from removing this: https://github.com/esl/MongooseIM/pull/4451/files#diff-e0e1d2948b30a346140efe9c7750bef22c779f6089a5f165e1385e05813bd374L1122.

This code was here as such because `mongoose_transport` would need to distinguish between s2s and component connections, in order to trigger the right instrumentation event. But components don't use `mongoose_transport` anymore, that was changed in the previous PRs but this remnant was left :)